website/config/routes.js

```diff
@@ -208,6 +208,14 @@ module.exports.routes = {
     },
   },
 
+  'GET /admin/email-preview': {
+    action: 'admin/view-email-templates'
+  },
+
+  'r|/admin/email-preview/((.+))$|': {
```

Comment:
I think this could be `GET /admin/email-preview/*`  (no regex route required).  Maybe I'm missing something